src/com/amazon/corretto/crypto/provider/AccessibleByteArrayOutputStream.java

```diff
@@ -7,6 +7,10 @@
 import java.util.Arrays;
 
 class AccessibleByteArrayOutputStream extends OutputStream implements Cloneable {
+  private static final int MAX_OVERSIZED_THRESHOLD = 1024;
+  private int timesOversized;
+  private int timesOversizedThreshold;
```

Comment:
What would be the initial value of the threshold? These need to be initialized somewhere.

Reply:
Here it would start at 0, which is fine by me, but also any small value would work (1, 2, 4).

Reply:
Actually 0 is a mistake here, since I'm doubling it to grow it. It must have been set to 1 in my testing and somehow I messed it up when committing. I'll fix it if we decide to use it.